cast/tests/e2e/helpers/devnet.rs

```diff
@@ -46,6 +45,10 @@ fn start_devnet() {
             std::process::exit(1);
         }
     }
+
+    let rt = Runtime::new().expect("Could not instantiate Runtime");
+    rt.block_on(declare_simple_balance_contract());
+    rt.block_on(deploy_simple_balance_contract());
```

Comment:
I suppose declare/deploy will work on different/seeded contracts, right? in that case, this is fine. otherwise, let's adjust to Karol's comment

Reply:
Deploy tests are not compromised because we can use different salt for deployment

Reply:
In the case of Declare, we will have to create some script which temporarily modifies the balance contract (eg. adds a character to one of the methods). It will change the class_hash